ansible/configs/ocp4-cluster/default_vars_osp.yml

```diff
@@ -29,7 +29,7 @@ ansible_user: cloud-user
 remote_user: cloud-user
 
 # The domain that you want to add DNS entries to
-osp_cluster_dns_zone: blue.osp.opentlc.com
+cluster_dns_zone: blue.osp.opentlc.com
```

Comment:
I think we can delete those since they now come from the secret.

Reply:
cool